src/builders/residfp-builder/residfp/WaveformCalculator.cpp

```diff
@@ -194,7 +194,9 @@ matrix_t* WaveformCalculator::buildTable(ChipModel model)
         wftable[6][idx] = calculateCombinedWaveform(cfgArray[2], 6, idx);
         wftable[7][idx] = calculateCombinedWaveform(cfgArray[3], 7, idx);
     }
-
+#ifdef HAVE_CXX11
+    return &(CACHE.emplace_hint(lb, cw_cache_t::value_type(cfgArray, wftable))->second);
+#endif
```

Comment:
Would be better to have an #else/#endif to avoid dead code

Reply:
To be more precise, if HAVE_CXX11 is defined there would be two return statements, so the second one should be included in an #else/#endif block.